recipes/install_binary.rb

```diff
@@ -8,9 +8,11 @@
 
 include_recipe 'ark'
 
+node.default['packer']['zipfile'] = "packer_#{node['packer']['version']}_" \
+  "#{node['os']}_#{node['packer']['arch']}.zip"
```

Comment:
I found an article that illustrates why setting derived attributes inside an attributes file can have unintended consequences. This is why it's better, in this particular case, to set the default in the recipe and not in the attributes/default.rb file.

https://christinemdraper.wordpress.com/2014/10/06/avoiding-the-possible-pitfalls-of-derived-attributes/